package.json

```diff
@@ -12,10 +12,13 @@
     "postinstall": "cd packages/react-instantsearch && npm install",
     "doctoc": "doctoc --maxlevel 1 README.md CONTRIBUTING.md",
     "test": "(cd packages/react-instantsearch && npm test) && npm run lint && NODE_ENV=production DOCS_MOUNT_POINT=/instantsearch.js/react/ DOCS_DIST=docs/react/ npm run docs:build && NODE_ENV=production npm run build",
-    "dev": "cd packages/react-instantsearch && npm run dev"
+    "dev": "cd packages/react-instantsearch && npm run dev",
+    "storybook": "start-storybook -p 6006",
```

Comment:
Do you use storybook heavily while developing here? (I believe yes). If so, what are we missing in our documentation website to not have to use storybook at all while developing? Maybe widgets previews with easy save/update?

I am not sure we want to have to ways to develop/try widgets on react-instantsearch. The current state of the doc dev flow is not ready compared to storybook but ultimately I would only keep one (the dev website).

What's your POV on this subject? Do you think it would be great to use storybook aside? I am really not into it so I may be missing a lot of possibilities that storybook offers that would complement well our own dev flow and library.

Maybe we could even integrate it in the dev website flow and documentation website output? Do  you know any library doing so for exhaustivity maybe along with ease of try/see?